### DISCUSSION TITLE
How to create a migration file directly without passing the path ?

### DISCUSSION THREAD

**Discussion by mohammedalbehairy** [NarHakobyan/awesome-nest-boilerplate#309] [General]
When running the **"migration:create"** script i got an error, but when i removed (  **-d ormconfig**  )  from the script and passed the target path it working fine.
But this will force me to add the path with every migration create, I need it to get the path directly from **ormconfig**

**Comment by NarHakobyan** [comment]
It's not possible, typeorm changed their settings for their cli
